Moduel3_4/Moduel3_4/Program.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+
+namespace Moduel3_4
+{
+    class Program
+    {
+        static void Main()
+        {
+            int numberFromUser = GetNumberFromUser("Enter the number:");
```

Comment:
what happen if i write negative number for example -123? I suppose that you will get in output 321-